### PR TITLE
Update reports.adoc

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/reports.adoc [hunk 1]
```diff
@@ -6,11 +6,13 @@ endif::[]
 *https://www.mulesoft.com/anypoint-pricing[This feature requires a Titanium subscription.]*
 
 == About Reports
-Reports allow you to view a list of all the CloudHub apps running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
+Reports allow you to view a list apps of running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
```
Comment (ignaciovazqueza) [inline modules/ROOT/pages/reports.adoc:9]: Shouldn't it be "of apps" instead?

Comment (sarathecoubian) [reply modules/ROOT/pages/reports.adoc:9]: Yes. My brain is not working. Thank you 😂

Comment (sarathecoubian) [reply modules/ROOT/pages/reports.adoc:9]:
```suggestion
Reports allow you to view a list of apps running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
```

File: modules/ROOT/pages/reports.adoc [hunk 2]
```diff
@@ -6,11 +6,13 @@ endif::[]
 *https://www.mulesoft.com/anypoint-pricing[This feature requires a Titanium subscription.]*
 
 == About Reports
-Reports allow you to view a list of all the CloudHub apps running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
+Reports allow you to view a list of apps running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
 
 From any of the *Reports* tabs, you can:
 
-* Navigate to reports that display information about requests, performance, failures, and CPU and memory utilization.
+* Navigate to reports that display information:
```
Comment (sarathecoubian) [inline modules/ROOT/pages/reports.adoc:13]:
```suggestion
* Navigate to reports that display the following information:
```

File: modules/ROOT/pages/reports.adoc [hunk 3]
```diff
@@ -6,11 +6,13 @@ endif::[]
 *https://www.mulesoft.com/anypoint-pricing[This feature requires a Titanium subscription.]*
 
 == About Reports
-Reports allow you to view a list of all the CloudHub apps running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
+Reports allow you to view a list of apps running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
 
 From any of the *Reports* tabs, you can:
 
-* Navigate to reports that display information about requests, performance, failures, and CPU and memory utilization.
+* Navigate to reports that display the following information:
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/reports.adoc:13]: Maybe this instead?
```suggestion
* View these reports:
```

File: modules/ROOT/pages/reports.adoc [hunk 4]
```diff
@@ -6,11 +6,13 @@ endif::[]
...
+** Requests reports, Performance reports, and Failures reports are available for all deployment types.
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/reports.adoc:14]: If "Requests", "Performance", and "Failures" are in the UI, I would make these bold:
```
** All deployment types provide *Requests* reports, *Performance* reports, and *Failures* reports.
```
If they're not, I would change the capitalization and plurals slightly:
```
** All deployment types provide request reports, performance reports, and failure reports.
```

File: modules/ROOT/pages/reports.adoc [hunk 5]
```diff
@@ -6,11 +6,13 @@ endif::[]
...
+** CPU Utilization reports and Memory Utilization reports are available only for CloudHub deployments.
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/reports.adoc:15]: Similarly, if "CPU Utilization" and "Memory Utilization" are in the UI:
```
** CloudHub deployments provide *CPU Utilization* reports and *Memory Utilization* reports.
```
And if not:
```
** CloudHub deployments provide CPU utilization reports and memory utilization reports.
```

Comment (sarathecoubian) [reply modules/ROOT/pages/reports.adoc:15]:
```suggestion
** CloudHub deployments provide *CPU Utilization* reports and *Memory Utilization* reports.
```

File: modules/ROOT/pages/reports.adoc (hunk 4 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/reports.adoc:14]:
```suggestion
** All deployment types provide *Requests* reports, *Performance* reports, and *Failures* reports.
```

**Pull request merged**